src/pages/[platform]/build-a-backend/graphqlapi/customize-authorization-rules/index.mdx

```diff
@@ -672,12 +692,11 @@ userPools:owner:owner
 
 ### Use IAM authorization within the AppSync console
 
-IAM-based `@auth` rules are scoped down to only work with Amplify-generated IAM roles. To access the GraphQL API with IAM authorization within your AppSync console, you need to explicitly allow list the IAM user's name.
 
 <BlockSwitcher>
 <Block name="Amplify CLI">
 
-Add the allow-listed IAM users by adding them to `amplify/backend/api/<your-api-name>/custom-roles.json`. (Create the `custom-roles.json` file if it doesn't exist). Append the `adminRoleNames` array with the IAM role or user names:
+IAM-based `@auth` rules are scoped down to only work with Amplify-generated IAM roles. To access the GraphQL API with IAM authorization within your AppSync console, you need to explicitly allow list the IAM user's name. Add the allow-listed IAM users by adding them to `amplify/backend/api/<your-api-name>/custom-roles.json`. (Create the `custom-roles.json` file if it doesn't exist). Append the `adminRoleNames` array with the IAM role or user names:
```

Comment:
Does this paragraph make sense ?

All they need to do is to set `authorizationModes.iamConfig.enableIamAuthorizationMode = true` and attach appropriate IAM policy to any non-cognito IAM principal. 

If we're deprecating `adminRoleNames` we should write this section from scratch. 

Or perhaps delete this block and just mention in section below that any non-cognito IAM principal would work.

Reply:
This section is specific to the Gen 1 CLI block of the docs, where they still have the old behaviro.